### PR TITLE
Bug fix: Healing dead players with light orbs

### PR DESCRIPTION
**Pull request by JordiRomagosa**
Should fix the bug where the players can take light orbs while dead.

I'm sorry but couldn't test the multiplayer properly because I lack a controller. Please confirm if it works properly.

### CODE REVIEW

**Review by Dartemiss — COMMENTED** [review]
Actually you can test it controlling just one of them, u kill the one that u controll and u just kill enemies with the debug feature "Kill all the enemies" and try to collect them.

File: Scripts/src/Script/LightOrb.cpp [hunk 1]
```diff
@@ -172,17 +189,23 @@ void LightOrb::Spawn(const float3& spawn_position)
 	exterior_texture_light_orb->Play();
 }
 
-GameObject* LightOrb::GetClosestPlayer() const
+Player* LightOrb::GetClosestAlivePlayer() const
 {
-	float distance_closest_player = player_1->player_go->transform.GetGlobalTranslation().Distance(owner->transform.GetGlobalTranslation());
-	GameObject* closest_player = player_1->player_go;
+	float distance_closest_player = 1000;
```
Comment (Dartemiss) [inline Scripts/src/Script/LightOrb.cpp:194]: 1000.f :P CS

**Review by Dartemiss — APPROVED** [review]
LGTM

**Comment by JordiRomagosa** [comment]
> 
> 
> Actually you can test it controlling just one of them, u kill the one that u controll and u just kill enemies with the debug feature "Kill all the enemies" and try to collect them.

Yes, that's true. But I can't test it with both (not only controllin p1, also needs to be tested controlling p2)

**Review by marcorod94 — COMMENTED** [review]
Nice job

**Review by marcorod94 — APPROVED** [review]
Nice job

**Pull request merged**